README.md

```diff
@@ -190,17 +190,40 @@ Get help.
 
 ### Hooks
 
-Hooks are event handlers and respond to Slack RTM API [events](https://api.slack.com/events), such as [hello](lib/slack-ruby-bot/hooks/hello.rb) or [message](lib/slack-ruby-bot/hooks/message.rb). You can implement your own by extending [SlackRubyBot::Hooks::Base](lib/slack-ruby-bot/hooks/base.rb).
+Hooks are event handlers and respond to Slack RTM API [events](https://api.slack.com/events), such as [hello](lib/slack-ruby-bot/hooks/hello.rb) or [message](lib/slack-ruby-bot/hooks/message.rb). You can implement your own in a couple of ways:
+
+
+#### Implement and register a Hook Handler
```

Comment:
hummmmm I capitalise concepts, not so much common actions (verbs) . But that's me :) , pick one (capitalise all or none), I'll change it accordingly